pkg/api/routes.go

```diff
@@ -72,6 +72,18 @@ func (h *Handler) GetRoutes() []Route {
 			h.GetCrushMap,
 		},
 		{
+			"CreateFileSysetm",
+			"POST",
+			"/file",
```

Comment:
i think we need a 1-pager for our API so we can establish better patterns. But /filesystem does make sense for now, i'll update.